src/website/src/pages/ec/components/index.mdx

```diff
@@ -30,6 +30,7 @@ import TimelineThumbnail from './timeline/ec_comp_timeline.svg';
 
 import HeroBannerThumbnail from './banners/hero-banner/ec_comp_hero_banner.svg';
 import PageBannerThumbnail from './banners/page-banner/ec_comp_page_banner.svg';
+import CarouselThumbnail from './carousel/ec_comp_carousel.svg';
```

Comment:
this icon is the the correct one. I assume that we didn't get the final icon yet